rllab/misc/logger.py

```diff
@@ -198,8 +201,10 @@ def dump_tabular(*args, **kwargs):
         # Also write to the csv files
         # This assumes that the keys in each iteration won't change!
         for tabular_fd in list(_tabular_fds.values()):
-            writer = csv.DictWriter(tabular_fd, fieldnames=list(tabular_dict.keys()))
-            if wh or (wh is None and tabular_fd not in _tabular_header_written):
+            writer = csv.DictWriter(
+                tabular_fd, fieldnames=list(tabular_dict.keys()))
+            if wh or (wh is None
+                      and tabular_fd not in _tabular_header_written):
```

Comment:
re-formated to these lines.

Reply:
I'm confused. Did you need to modify the logger.py to add dm_control support? If so, you can reformat the whole file. Otherwise, just leave logger.py out.